### DISCUSSION TITLE
Intensity Limit?

### DISCUSSION THREAD

**Discussion by aDenos24** [Nesvilab/FragPipe-Analyst-doc#70] [General]
I was wondering if Fragpipe has an intensity minimum. I am noticing lots of holes in my data, with the lowest intensity value being about 170,000. Is there a parameter that I can allow fragpipe to go lower than that? At least to 10,000 or 1,000?

**Comment by fcyu** [comment]
There is no intensity thresholding in FragPipe. But there are other quality filtering criteria, such as min isotopes and min scans.

BTW, please send the FragPipe related questions to https://github.com/Nesvilab/FragPipe/issues in the future.

Best.

Fengchao
